src/layers/ConstructionLayer/ConstructionLayer.js

```diff
@@ -19,65 +20,70 @@ import { unByKey } from 'ol/Observable';
  */
 class ConstructionLayer extends VectorLayer {
   constructor(options = {}) {
-    const olLayer = new OLVectorLayer({
-      style: (f, r) => this.style(f, r),
-      source: new OLVectorSource({
-        format: new GeoJSON(),
-        loader: () => {
-          fetch(
-            `${this.geoServerUrl}?` +
-              'service=WFS&version=1.0.0&request=GetFeature&' +
-              'typeName=trafimage:bahnausbauten&' +
-              'srsName=EPSG:3857&' +
-              'outputFormat=application/json',
-          )
-            .then(data => data.json())
-            .then(data => {
-              const format = new GeoJSON();
-              const features = format.readFeatures(data);
-              this.getSource().clear();
-              this.getSource().addFeatures(features);
-            });
-        },
-      }),
-      minResolution: options.minResolution,
-      maxResolution: options.maxResolution,
+    const singleLayer = new OLVectorLayer({
+      style: (f, r) => this.styleSingle(f, r),
+      maxResolution: 305.748113141,
+    });
+
+    const clusterLayer = new OLVectorLayer({
+      style: (f, r) => this.styleCluster(f, r),
+      minResolution: 305.748113141,
```

Comment:
It makes sens to have one variable for the 2 305.748113141 values

Reply:
Done